masterlist.yaml

```
@@ -5446,8 +5446,9 @@ plugins:
         itm: 130
         udr: 218
 
-  - name: 'Landscape and Water Fixes.esp'
+  - name: '(Skyrim )?Landscape and Water Fixes( - Patch - | - Light Sources)?(CACO|ELFX|Relighting Skyrim)?.esp'
```

I'd probably just use the following as it will work for other patches.

```suggestion
  - name: '(Skyrim )?Landscape and Water Fixes.*\.esp'
```